railgun/keps/0003-kic-kubebuilder-re-architecture.md

```diff
@@ -18,8 +19,19 @@ status: implementable
   - [Test Plan](#test-plan)
   - [Graduation Criteria](#graduation-criteria)
 - [Implementation History](#implementation-history)
+- [Alternatives](#alternatives)
 <!-- /toc -->
 
+## Release Signoff Checklist
```

Comment:
We already have a process for issues blocking other issues - putting a "blocker" label with a reference to blocker issues in the GH issue tracker. My feeling is that the release signoff checklist duplicates that (in a way that will be hidden from our typical work planning practices).

Unless you see a need for such checklists in KEPs, I recommend that we stick with the existing "blocker process" rather than defining such a checklist here. I'm open to hearing your thoughts here, though :thinking:

Reply:
I would consider this a reference, and would be happy to remove the checklist portion of it if you think that improves it? It's mostly about posterity.

Reply:
Removed and made into NOTES